lightningd/log.c

```diff
@@ -53,10 +53,16 @@ static void log_default_print(const char *prefix,
 			      bool continued,
 			      const char *str, void *arg)
 {
+	struct timeval t;
+	gettimeofday(&t, NULL);
+	char iso8601_s[strlen("YYYY-mm-ddTHH:MM:SS.nnnZ") + 1];
```

Comment:
Should be a global `const size_t` to avoid repeated `strlen` calls.

Reply:
Gcc will optimize. But sizeof() is even better...